llama.cpp

```diff
@@ -6068,13 +6175,15 @@ static int llama_decode_internal(
         }
     }
 
+#if 0
```

Comment:
This is probably some leftover - should not be removed

Reply:
I think only @niansa knows why this was originally commented out.